tests/ci/notebooks_unit_tests_tf2.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we need a separate test pipeline for TF2. Once we upgrade the TF version in setup.py the current pipeline will switch to version 2. So you don't need this file.

Reply:
+1

Reply:
removed this file.